fpm/recipes/ruby-2.7.3/recipe.rb

```diff
@@ -0,0 +1,31 @@
+class RbenvRuby273 < FPM::Cookery::Recipe
+  homepage 'https://www.ruby-lang.org/'
+  name 'rbenv-ruby-2.7.3'
+  version '1'
+
+  source 'https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.3.tar.gz'
+  sha256 '8925a95e31d8f2c81749025a52a544ea1d05dad18794e6828709268b92e55338'
+
+  maintainer 'GOV.UK <govuk-dev@digital.cabinet-office.gov.uk>'
+  license 'Ruby'
```

Comment:
How come license is ruby here and not MIT?

Reply:
I just copied it from the other recipes 🤷‍♂️